contracts/deploy/035_multiple_rewards_strategies.js

```diff
@@ -0,0 +1,179 @@
+const { deploymentWithProposal, log } = require("../utils/deploy");
+
+module.exports = deploymentWithProposal(
+  { deployName: "035_convex_rewards", forceDeploy: true },
+  async ({
+    assetAddresses,
+    deployWithConfirmation,
+    ethers,
+    getTxOpts,
+    withConfirmation,
+  }) => {
+    const { deployerAddr, governorAddr } = await getNamedAccounts();
+    const sDeployer = await ethers.provider.getSigner(deployerAddr);
+    const dVaultAdmin = await deployWithConfirmation("VaultAdmin");
+    const dVaultCore = await deployWithConfirmation("VaultCore");
+    log("Deployed VaultAdmin and VaultCore...");
+
+    // Current contracts
+    const cVaultProxy = await ethers.getContract("VaultProxy");
+    const cVaultAdmin = await ethers.getContractAt(
+      "VaultAdmin",
+      cVaultProxy.address
+    );
+    const cVaultCore = await ethers.getContractAt(
+      "VaultCore",
+      cVaultProxy.address
+    );
+    const cVault = await ethers.getContractAt("Vault", cVaultProxy.address);
+
+    // Deployer Actions
+    // ----------------
+
+    // 1. Deploy new implementation
+    const dConvexStrategyImpl = await deployWithConfirmation(
+      "ConvexStrategy",
+      undefined,
+      undefined,
+      true // Disable storage slot checking, we are renaming variables in InitializableAbstractStrategy.
+    );
+    const dCompoundStrategyImpl = await deployWithConfirmation(
+      "CompoundStrategy",
+      undefined,
+      undefined,
+      true // Disable storage slot checking, we are renaming variables in InitializableAbstractStrategy.
+    );
+    const dAaveStrategyImpl = await deployWithConfirmation(
+      "AaveStrategy",
+      undefined,
+      undefined,
+      true // Disable storage slot checking, we are renaming variables in InitializableAbstractStrategy.
+    );
+    const dThreePoolStrategyImpl = await deployWithConfirmation(
```

Comment:
We don't have three pool deployed to production right now. We can skip deploying an upgrade.

Reply:
Tnx removed